jest.config-standalone.js

```diff
@@ -1,9 +1,12 @@
 module.exports = {
   preset: 'ts-jest',
-  globals: {
-    'ts-jest': {
-      tsconfig: 'tsconfig.json',
-    },
+  transform: {
+    '^.+\\.tsx?$': [
+      'ts-jest',
+      {
+        tsconfig: 'tsconfig.json',
+      },
+    ],
```

Comment:
Why this change?

Reply:
For a moment I thought it was related to the first issue I had with axios: 
![image](https://github.com/DataDog/datadog-ci/assets/9317502/ac23d51c-2b1b-406b-bdc7-4d65cf31eb6a)

Turns out it was _really_ just a warning, but it's still a good thing to fix 😄